cmd/opm/render/cmd.go

```diff
@@ -44,6 +46,29 @@ func NewCmd() *cobra.Command {
 			// returned from render.Run and logged as fatal errors.
 			logrus.SetOutput(ioutil.Discard)
 
+			skipTLSVerify, useHTTP, err := util.GetTLSOptions(cmd)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			cacheDir, err := action.NewRenderCacheDir()
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			reg, err := containerd.NewRegistry(
+				containerd.WithCacheDir(cacheDir),
+				containerd.SkipTLSVerify(skipTLSVerify),
+				containerd.WithPlainHTTP(useHTTP),
+				containerd.WithLog(action.NullLogger()),
+			)
+			if err != nil {
+				log.Fatalf("error creating containerd registry: %v", err)
```

Comment:
really minor nit just to shorten the length of the error message. the context of the log message should make it obvious that an error occurred.
```suggestion
				log.Fatalf("create containerd registry: %v", err)
```